packages/react-components/eslint-plugin-react-components/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33214&commitId=becf4367d8617693476af97bcce16a185c732e99&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>